kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/ListTopicConfigsResponse.java

```diff
@@ -21,16 +21,16 @@
 import java.util.StringJoiner;
 
 /**
- * Response body for {@code GET /v3/clusters/<clusterId>/topics/<topicName>/configurations}
+ * Response body for {@code GET /v3/clusters/<clusterId>/topics/<topicName>/configs}
  * requests.
  */
-public final class ListTopicConfigurationsResponse {
+public final class ListTopicConfigsResponse {
```

Comment:
as noted before, technically incorrect, but acceptable imo and terse and not public. either as is or `ListTopicConfigPropertiesResponse`

Reply:
i'm starting to prefer `ListTopicConfigPropertiesResponse`, even though very long.